java/plantuml/Portfile

```diff
@@ -4,21 +4,21 @@ PortSystem              1.0
 PortGroup               github 1.0
 PortGroup               java 1.0
 
-github.setup            plantuml plantuml 1.2020.11 v
+github.setup            plantuml plantuml 1.2021.4 v
 revision                0
 categories              java editors
 platforms               darwin
 supported_archs         noarch
 license                 GPL-3+
-maintainers             nomaintainer
+maintainers             outlook.de openmaintainer
```

Comment:
This doesn't look right... `outlook.de` isn't an email address in the format MacPorts expects...

Reply:
Perhaps I accidentally pressed the cancellation in the vim. Thanks!